cfpb/consumerfinance.gov
# [TDP survey] Fix error handling and handle beta banner

Fixes error handling that was broken due to markup changes. Also makes sure the beta site banner doesn't accidentally get selected by querySelector().

---

## Changes

- TDP survey error handling uses more specific CSS selector
- TDP survey error handling correctly shows the legend text in the error box

## How to test this PR

1. Open https://beta.consumerfinance.gov/consumer-tools/educator-tools/youth-financial-education/survey/3-5/p1/
2. Click [Next] at the bottom to trigger form errors
3. Verify you're presented a list of unanswered questions at the top.

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
